sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -363,44 +363,121 @@ const alwaysBeDeletedRule = "alwaysbedeletedrule";
         should.equal(response._response.status, 200);
       });
 
-      it(`Get on non-existent ${entityType} entity returns empty response`, async () => {
+      it(`Get on non-existent ${entityType} entity throws an error`, async () => {
         let response;
-
-        let skipTest = false;
         switch (entityType) {
           case EntityType.QUEUE:
+            try {
+              await deleteEntity(entityType, alwaysBeExistingQueue);
```

Comment:
Can we not skip the delete step by using a randomly generated unique id as name for the get operation?

Reply:
Using simple "notexisting" as entity name should suffice